homework/jenya_s/homework4.py

```diff
@@ -0,0 +1,25 @@
+def find_most_frequent(text):
+
+    # making a new string in lowercase
+    # and replacing punctuation marks in it with spaces
+    sub_str = text.lower()
+    translation_table = dict.fromkeys(map(ord, ",.:;!?-"), " ")
+    sub_str2 = sub_str.translate(translation_table)
+
+    # creating a list of separate words and getting rid of spaces
+    lst = [x for x in sub_str2.split(" ") if x!=""]
+    
+    # forming a list of most frequent words
+    final_lst = [i for i in lst if lst.count(i) >= (max(lst.count(i) for i in lst))]
```

Comment:
move max value calculating into separate variable, it decreases complexity of expression